pages/docs/tools/generator/hooks.md

```diff
@@ -10,7 +10,7 @@ The following types of hooks are currently supported:
 |---|---|---|---|
 | `generate:before` | Called after registration of all filters and before the generator starts processing of the template. | void : Nothing is expected to be returned. | [The generator instance](https://github.com/asyncapi/generator/blob/master/docs/api.md)
 | `generate:after` | Called at the very end of the generation. | void : Nothing is expected to be returned. | [The generator instance](https://github.com/asyncapi/generator/blob/master/docs/api.md)
-| `setFileTemplateName ` | Called right before saving a new file generated by [file template](./file-templates.md). | string : a new filename for the generator to use for the file template. | [The generator instance](https://github.com/asyncapi/generator/blob/master/docs/api.md) and object in the form of `{ "originalFilename" : string }`
```

Comment:
Oooh wait, this needs to be a PR in the `/generator` repo, huh? 😅 @derberg dammit.. haha 

I am actually also talking to @Florence-Njeri about other broken links I found there and logged in https://github.com/asyncapi/website/issues/2367, so I guess this should go under that instead.

Reply:
Ok lukasz said we can merge and it's good.